api/src/test/java/com/thesurvey/api/controller/SurveyControllerTest.java

```diff
@@ -226,6 +226,37 @@ void testUpdateSurvey() throws Exception {
             LocalDateTime.parse(content.getString("endedDate")));
     }
 
+    @Test
+    void testGetAllSurvey() throws Exception {
+        // given
+        for (int i = 1; i < 16; i++) { // create test 15 surveys
+            SurveyRequestDto surveyRequestDto = SurveyRequestDto.builder()
+                .title("This is test survey title " + i)
+                .description("This is test survey description")
+                .startedDate(LocalDateTime.now(ZoneId.of("Asia/Seoul")).plusDays(1))
+                .endedDate(LocalDateTime.now(ZoneId.of("Asia/Seoul")).plusDays(2))
+                .certificationTypes(List.of(CertificationType.GOOGLE))
+                .questions(List.of(questionRequestDto))
+                .build();
+            mockCreateSurvey(surveyRequestDto);
+        }
+
+        // when
+        MvcResult result = mockMvc.perform(get("/surveys")
+                .param("page", "1")
```

Comment:
![array](https://user-images.githubusercontent.com/63793178/233555489-6e36a967-ebd2-4670-a5ad-4110704dcdac.png)

Reply:
And this starts from `1`!